app/mailers/team_leader_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class TeamLeaderMailer < ApplicationMailer
+  def new_project_created(team_leader, project)
+    template_mail(
+      "ea4f72e4-f5bb-4b1a-b5f9-a94cc1840353",
```

Comment:
Hmm, will all the template IDs get unwieldly as we add more? Should they be environment variables?

Reply:
I think this is good, just thinking ahead and about how best to handle these IDs, this one is encapsulated in the `TeamLeaderMailer` which I like, but I can see this coming apart when we have to send to multiple roles...

However we tackle this one sets the tone for those that follow (in my experience) so this is our chance to lay some best practice foundaitons.